src/components/Square.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Square = ({ image, idx, fabric, debug, quiltSectionWidth }) => {
+  // if even number wide, true.
+  // if not even number wide, make even every other row.
+  //   const isEven =
+  //     quiltSectionWidth % 2 === 0 ||
+  //     Math.floor(idx / quiltSectionWidth) % 2 === 0;
+  return (
+    <div
+      css={[
+        {
+          overflow: "hidden",
+          position: "relative",
+          clipPath: "polygon(0% 0%, 100% 0%, 100% 100%, 0% 100%)",
+          width: `calc(125vw / ${quiltSectionWidth})`,
```

Comment:
Since we don't need squares to overlap each other, we can match the vw measurement used on the <article /> container.

```suggestion
width: `calc(94vw / ${quiltSectionWidth})`,
```